src/applications/hca/containers/ConfirmationPage.jsx

```diff
@@ -27,45 +27,35 @@ export class ConfirmationPage extends React.Component {
 
     let emailMessage;
 
-    const title = 'Your claim is pending';
```

Comment:
there shouldn't be any changes to this file now

Reply:
Re-restored original and pushed up again.  Now no longer listed in Files-changed.